src/content/en/tools/puppeteer/_toc.yaml

```diff
@@ -0,0 +1,104 @@
+      - title: v0.13.0
+        section:
+          - title: Docs
+            path: 'https://github.com/GoogleChrome/puppeteer/blob/v0.13.0/docs/api.md'
+            status: external
+          - title: Release Notes
+            path: 'https://github.com/GoogleChrome/puppeteer/releases/tag/v0.13.0'
+            status: external
+      - title: v0.12.0
+        section:
+          - title: Docs
+            path: 'https://github.com/GoogleChrome/puppeteer/blob/v0.12.0/docs/api.md'
+            status: external
+          - title: Release Notes
+            path: 'https://github.com/GoogleChrome/puppeteer/releases/tag/v0.12.0'
+            status: external
+      - title: v0.11.0
+        section:
+          - title: Docs
+            path: 'https://github.com/GoogleChrome/puppeteer/blob/v0.11.0/docs/api.md'
+            status: external
+          - title: Release Notes
+            path: 'https://github.com/GoogleChrome/puppeteer/releases/tag/v0.11.0'
+            status: external
+      - title: v0.10.2
+        section:
+          - title: Docs
+            path: 'https://github.com/GoogleChrome/puppeteer/blob/v0.10.2/docs/api.md'
+            status: external
+          - title: Release Notes
+            path: 'https://github.com/GoogleChrome/puppeteer/releases/tag/v0.10.2'
+            status: external
+      - title: v0.10.1
+        section:
+          - title: Docs
+            path: 'https://github.com/GoogleChrome/puppeteer/blob/v0.10.1/docs/api.md'
+            status: external
+          - title: Release Notes
+            path: 'https://github.com/GoogleChrome/puppeteer/releases/tag/v0.10.1'
+            status: external
+      - title: v0.10.0
+        section:
+          - title: Docs
+            path: 'https://github.com/GoogleChrome/puppeteer/blob/v0.10.0/docs/api.md'
+            status: external
+          - title: Release Notes
+            path: 'https://github.com/GoogleChrome/puppeteer/releases/tag/v0.10.0'
+            status: external
+      - title: v0.9.0
+        section:
+          - title: Docs
+            path: 'https://github.com/GoogleChrome/puppeteer/blob/v0.9.0/docs/api.md'
+            status: external
+          - title: Release Notes
+            path: 'https://github.com/GoogleChrome/puppeteer/releases/tag/v0.9.0'
+            status: external
+  - title: Examples
+    path: /web/tools/puppeteer/examples
+  - heading: Resources
+  - title: Debugging tips
+    path: /web/tools/puppeteer/debugging
+  - title: Troubleshooting
+    path: /web/tools/puppeteer/troubleshooting
+  - title: Headless Chrome
+    style: accordian
+    section:
+      - title: Getting Started with Headless Chrome
+        path: /web/updates/2017/04/headless-chrome
+      - title: Automated testing with Headless Chrome
+        path: /web/updates/2017/06/headless-karma-mocha-chai
+  - title: DevTools Protocol APIs
+    path: 'https://chromedevtools.github.io/devtools-protocol/'
+  - title: Try Puppeteer!
```

Comment:
If you create a project yaml you can stuff a tonne of these links in the footer.

Reply:
done

Reply:
actually, going to hold off on this for now.